charts/consul/values.yaml

```diff
@@ -1144,6 +1144,58 @@ server:
     # @type: string
     caCert: null
 
+  # [Enterprise Only] Added in Consul 1.8, the audit object allow users to enable auditing 
+  # and configure a sink and filters for their audit logs
+  auditLogs:
```

Comment:
Would it be better to use explicit values instead of an array? Probably easier for a user to provide config but then we'd have to think about defaults.